package.json

```diff
@@ -166,6 +168,7 @@
     "matrix-web-i18n": "github:matrix-org/matrix-web-i18n",
     "react-test-renderer": "^17.0.2",
     "rimraf": "^3.0.2",
+    "rrweb": "^0.9.9",
```

Comment:
this dependency might be scary to users "record and replay the web"

Reply:
Well, I guess it's a dev dependency. Although the reason it's here is to satisfy posthog's type definitons' need to import  it, it does seems like having a recording dependency for dev might be defensible e.g. for debugging or integration testing

Reply:
See if you prefer https://github.com/matrix-org/matrix-react-sdk/pull/6495/commits/e4722ee4578dc7d06999c68b7e321731c598b1d8